python/ray/autoscaler/kubernetes/example-full.yaml

```diff
@@ -78,127 +74,86 @@ provider:
             # NOTE: If you're running multiple Ray clusters with services
             # on one Kubernetes cluster, they must have unique service
             # names.
-            name: ray-head
+            name: example-cluster-ray-head
         spec:
             # This selector must match the head node pod's selector below.
             selector:
-                component: ray-head
-            ports:
-                - protocol: TCP
-                  port: 8000
-                  targetPort: 8000
-
-      # Service that maps to the worker nodes of the Ray cluster.
-      - apiVersion: v1
-        kind: Service
-        metadata:
-            # NOTE: If you're running multiple Ray clusters with services
-            # on one Kubernetes cluster, they must have unique service
-            # names.
-            name: ray-workers
-        spec:
-            # This selector must match the worker node pods' selector below.
-            selector:
-                component: ray-worker
+                component: example-cluster-ray-head
             ports:
-                - protocol: TCP
-                  port: 8000
-                  targetPort: 8000
-
-# Kubernetes pod config for the head node pod.
-head_node:
-    apiVersion: v1
-    kind: Pod
-    metadata:
+                - name: client
+                  protocol: TCP
+                  port: 10001
+                  targetPort: 10001
+                - name: dashboard
+                  protocol: TCP
+                  port: 8265
+                  targetPort: 8265
+
+# Specify the pod type for the ray head node (as configured below).
+head_node_type: head_node
+# Specify the allowed pod types for this ray cluster and the resources they provide.
+available_node_types:
+  worker_node:
+    # Minimum number of Ray workers of this Pod type.
+    min_workers: 2
```

Comment:
can we change this to min_workers = 0 ?

Reply:
okidoke -- will just indicate in the docs how to test out autoscaling by raising min_workers